### PR TITLE
[WIP] Expiring auctions UI

### PR DESCRIPTION
**Pull request by gemfarmer**
⚠️ [WIP] ⚠️ – currently blocked by the viewmodel refactor

For #536

### CODE REVIEW

**Comment by gemfarmer** [comment]
@jessieay I will just work on styles for this PR until you are done refactoring. Lemme know when this has been refactored so I know that I can start refactoring the ViewModel and presenter stuff.

**Comment by gemfarmer** [comment]
Current status as of 7ce5a24

screen shot 2016-05-24 at 2 46 03 pm

**Comment by jessieay** [comment]
😍

**Comment by jessieay** [comment]
@gemfarmer now you have some nice merge conflicts -- new view model for index was merged!

**Comment by gemfarmer** [comment]
@jessieay fixed the merge conflicts. Now I need to comb through and make sure I don't have superfluous methods and partials :( . I'll let you know when it needs an extra set of 👀  ‼️

File: app/presenters/human_time.rb [hunk 1]
```diff
@@ -11,13 +11,13 @@ def initialize(time:, current_time: Time.current)
   def relative_start_time
```
Comment (jessieay) [inline app/presenters/human_time.rb]: we are using this method in a few places -- double check that this usage makes sense in all places? Otherwise, let's add these extra strings in the place where you want them and interpolate `relative_start_time`

File: app/views/auctions/labels/_time_remaining.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="issue-bid-deadline">
+  <% if auction.over? or auction.future? %>
```
Comment (jessieay) [inline app/views/auctions/labels/_time_remaining.html.erb]: the new refactor has us removing all conditionals from views and putting them in the view models -- can you do that here? If you'd like help, I am more than happy to pair.

File: app/view_models/auction_list_item.rb [hunk 3]
```diff
@@ -64,10 +76,10 @@ def item_footer_partial
   end
 
   def winning_bid_partial
-    if auction.bids.any?
```
Comment (jessieay) [inline app/view_models/auction_list_item.rb]: Let's push the conditionals into this method rather than using two partials here

File: db/schema.rb [hunk 4]
```diff
@@ -73,8 +73,8 @@
     t.datetime "created_at",                           null: false
     t.datetime "updated_at",                           null: false
     t.string   "email"
-    t.string   "github_login"
```
Comment (jessieay) [inline db/schema.rb]: git checkout on this file? seems unrelated to changeset

**Comment by gemfarmer** [comment]
Closing in favor of #616 ! ‼️
